src/metabase/public_settings.clj

```diff
@@ -71,6 +71,17 @@
   :type    :boolean
   :default true)
 
+(defsetting reset-token-ttl-h
+  (deferred-tru "Number of hours a password reset is considered valid.")
+  :visibility :settings-manager
```

Comment:
Per @cdeweyx's comment, if you change this to `:visibility :internal` it will be an env var-only setting automatically and can be set via `MB_RESET_TOKEN_TTL_H`. You can also get rid of the custom setter then.

Reply:
@noahmoss, apologies for the delay, have addressed the changes